apps/core/scenes/search/PlacePicker/PlaceItem.js

```diff
@@ -68,9 +78,16 @@ class PlaceItem extends React.Component<Props> {
 const styles = StyleSheet.create({
   container: {
     flexDirection: 'row',
-    alignItems: 'center',
+    alignItems: 'flex-end',
   },
-  text: {
+  iconContainer: {
+    paddingBottom: 8,
+  },
+  textContainer: {
+    flex: 1,
+    marginLeft: 5,
```

Comment:
I thought units were meant to be divisible by 4?

Reply:
True :) fixed